### ISSUE TITLE
users() also gets contacts

### ISSUE THREAD

**Issue by KuenzelIT** [Adldap2/Adldap2#517]
- Adldap2 Version: 8.1.3
- LDAP Type: Active Directory
- PHP Version: 7.1.14

### Description:
If I use the search functionality with 
```php
$query = Adldap::search()->users()->get()
```
, I'm also getting the contacts from my active directory. Is this intended? I assume no since there is a special contacts scope?

I'm currently filtering the contacts out with 
```php
return $users->reject(function ($item) {
      return $item instanceof Contact;
});
```

Thanks!

### Steps To Reproduce:
Use `$query = Adldap::search()->users()->get()`

**Comment by stevebauman** [comment]
Hi @KuenzelIT, this was originally intended, but this will be changed.

Let me explain:

Entries in ActiveDirectory can contain multiple object classes.

Since some ActiveDirectory contacts can contain the objectClass of `person`, the objectCategory of `person` **and** the objectClass of `contact` , they may be returned since they are contained in the same scope that is currently being queried for:

https://github.com/Adldap2/Adldap2/blob/master/src/Query/Factory.php#L157-L158

I'll add another `where` clause to reject the objectClass of `contact` to the `users()` scope so this doesn't occur.

I believe you can actually resolve this yourself now by doing the following:

```php
$users = $provider->search()->users()->whereNotContains('objectclass', 'contact')->get();
```

Can you give that a shot and let me know?

**Comment by KuenzelIT** [comment]
Hey,

thank you for the quick response.

Unfortunately this change leads to an empty list, not a single user is found with that `whereNotContains`. I cannot find out why right now.

**Comment by bochoven** [comment]
Hi @stevebauman, any ETA on the change you mentioned? I manage a project that relies on this excellent library that experiences the same issue: https://github.com/munkireport/munkireport-php/issues/1081

**Comment by ThomHurks** [comment]
@stevebauman Was this fix ever released? I just updated Adldap-laravel to 5.0.1 and adldap2 to 9.1.3 and I'm now seeing ```(!(objectclass=contact))``` being added to the search query, breaking user authentication completely.
